src/main/scala/wdl4s/expression/ValueEvaluator.scala

```diff
@@ -113,7 +113,30 @@ case class ValueEvaluator(override val lookup: String => WdlValue, override val
               case o: WdlObjectLike =>
                 o.value.get(rhs.getSourceString) match {
                   case Some(v:WdlValue) => Success(v)
-                  case None => Failure(new WdlExpressionException(s"Could not find key ${rhs.getSourceString}"))
+                  case None =>
```

Comment:
I don't complain about this as much as I used to but this sub-case is pretty lengthy to follow